### DISCUSSION TITLE
run applications with autodarkmode

### DISCUSSION THREAD

**Discussion by pho084** [AutoDarkMode/Windows-Auto-Night-Mode#432] [Q&A] [answered]
hi
first of thanks for this awesome app. i have a quick question/feature request: would it be possible to add an option to run external applications in line with autodarkmode schedule?

e.g. i want my power schedule to change between balanced during the day and maximum performance at night as soon as i log on. i could incorporate this with autodarkmode via a batch file triggering powercfg

cheers

**Comment by Armin2208** [comment, marked as answer]
we implemented support for running custom scripts. Further information [on the wiki page](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/wiki/How-to-add-custom-scripts).

**Reply by pho084** [reply]
maybe you could help me a bit more. so i tried editing scripts.yaml to change powerplans in windows in line with autodarkmode, like outlined below, unfortunately it does not work (i also tried it without the ".exe"). Any idea. thanks

Enabled: true
Component:
  TimeoutMillis: 10000
  Scripts:
  - Name: powerplanswitch
    Command: powercfg.exe
    ArgsLight: [-SETACTIVE 381b4222-f694-41f0-9685-ff5bb260df2e]
    ArgsDark: [-SETACTIVE 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c]
    AllowedSources: [Any]

**Reply by pho084** [reply]
nevermind, figured it out; 

i was missing the comma after the -SETACTIVE

now it all works, thanks again
